src/collector/mod.rs

```diff
@@ -422,6 +453,7 @@ impl Collector {
         // Now cleanup by removing all the data that is done for
         par_retain(&self.tracked_data.data, |data, _| {
             // Mark the new data as in use for now
+            // This stops us blinding data that was allocated during collection
```

Comment:
I think you meant “building” instead of “blinding” here? Not 100% sure, but it seems like a typo.